src/scene/constants.js

```diff
@@ -196,6 +204,42 @@ export var LIGHTTYPE_POINT = 1;
  */
 export var LIGHTTYPE_SPOT = 2;
 
+/**
+ * @constant
+ * @private
+ * @name pc.LIGHTSHAPE_PUNCTUAL
+ * @type {number}
+ * @description Infinitesimally small point light source shape.
+ */
+export var LIGHTSHAPE_PUNCTUAL = 0;
+
+/**
+ * @constant
+ * @private
+ * @name pc.LIGHTSHAPE_RECT
+ * @type {number}
+ * @description Rectangle shape of light source.
+ */
+export var LIGHTSHAPE_RECT = 1;
+
+/**
+ * @constant
+ * @private
+ * @name pc.LIGHTSHAPE_DISK
+ * @type {number}
+ * @description Disk shape of light source.
+ */
+export var LIGHTSHAPE_DISK = 2;
```

Comment:
I saw this and immediately thought it should be DISC. But then I read that DISK is American English. So let's stick with this. Although interesting to see Unity goes with Disc: https://docs.unity3d.com/Packages/com.unity.render-pipelines.high-definition@8.0/api/UnityEngine.Rendering.HighDefinition.AreaLightShape.html

Reply:
Using American English confuses fewer people - when I first added it, I added it as DISC - but changed it to DISK.